src/main/java/com/github/jreddit/parser/entity/imaginary/MixedListingElement.java

```diff
@@ -0,0 +1,6 @@
+package com.github.jreddit.parser.entity.imaginary;
+
+
+public interface MixedListingElement {
```

Comment:
Reason added.

Interface to abstract over the two possible elements in a mixed listing, namely a {@link Comment} or a {@link Submission} thing. If an object is of the type of this interface, it means that it <i>must</i> be either of the respective {@link Comment} or {@link Submission} class.